pkg/tsdb/loki/framing_test.go

```diff
@@ -72,10 +76,8 @@ func TestSuccessResponse(t *testing.T) {
 
 	for _, test := range tt {
 		t.Run(test.name, func(t *testing.T) {
-			runTest("testdata", test.filepath, test.query, ResponseOpts{metricDataplane: false, logsDataplane: false})
-			runTest("testdata_metric_dataplane", test.filepath, test.query, ResponseOpts{metricDataplane: true, logsDataplane: false})
-			runTest("testdata_logs_dataplane", test.filepath, test.query, ResponseOpts{metricDataplane: false, logsDataplane: true})
-			runTest("testdata_dataplane", test.filepath, test.query, ResponseOpts{metricDataplane: true, logsDataplane: true})
+			runTest("testdata_metric_dataplane", test.filepath, test.query, ResponseOpts{logsDataplane: false})
+			runTest("testdata_dataplane", test.filepath, test.query, ResponseOpts{logsDataplane: true})
```

Comment:
While we are at it, do we wanna rename these tests/directories to something more descriptive?

Reply:
Absolutely. Any suggestion?

Reply:
I got it. I'm new to this part of the codebase so I wasn't sure of the original intention.